tests/Commands/CacheSchedulerRunningTest.php

```diff
@@ -20,8 +20,10 @@ public function running_command_updates_cache()
 
         $this->app->register(HealthCheckServiceProvider::class);
 
-        $this->artisan('health-check:cache-scheduler-running');
-        
-        $this->assertTrue(true); // check command ran without error
+        $result = $this->artisan('health-check:cache-scheduler-running');
+
+        if ($result instanceof PendingCommand) {
```

Comment:
Can we use assertInstanceOf here? Otherwise we might end up with those PHPUnit warnings saying a test has no assertions

Reply:
No `assertInstanceOf` we can't add, but we can add `assertTrue`

Reply:
ahh, fair enough then